tests/test_flows.py

```diff
@@ -453,6 +453,46 @@ def test_combined_infra_metrics(self):
         self.check_log(self.repeater1, "agent", "Received TLV_TRANSMITTER_LINK_METRIC")
         self.check_log(self.repeater1, "agent", "Received TLV_RECEIVER_LINK_METRIC")
 
+    def test_client_capability_query(self):
+        sta_mac1 = '00:00:00:11:00:22'
+        sta_mac2 = '00:00:00:11:00:33'
+
+        self.gateway_ucc.dev_send_1905(self.mac_repeater1, 0x8009,
```

Comment:
A comment explaining the 1905 command being sent will help understand the flow better.